app/services/reports/doc_auth_drop_off_rates_per_sprint_report.rb

```diff
@@ -25,7 +25,8 @@ def generate_sprints(ret, date, today_date)
         transaction_with_timeout do
           start = date
           finish = date.next_day(14)
-          ret << Db::DocAuthLog::DropOffRates.new.call(fmt(start), fmt(finish))
+          ret << Db::DocAuthLog::BlanketDropOffRatesAllSpsInRange.new.
+                 call('Sprint', fmt(start), fmt(finish))
```

Comment:
What do you think about passing in Date objects and using `ActiveRecord::Base.connection.quote(date)` to escape them like we did in the other reports?